test/cmocka/src/debugability/macros.c

```diff
@@ -45,15 +45,20 @@
 static void test_debugability_macros_declare_log_entry(void **state)
 {
 	const char *macro_result = CAPTURE(_DECLARE_LOG_ENTRY(
-		LOG_LEVEL_CRITICAL, "Message", TRACE_CLASS_DMA, 1));
+		LOG_LEVEL_CRITICAL,
+		"Message",
+		TRACE_CLASS_DMA,
+		1
+	));
 	const char *should_be_eq =
 		"__attribute__((section(\".static_log.\""
 		" \"LOG_LEVEL_CRITICAL\"))) "
 		"static const struct { uint32_t level; uint32_t component_id; "
 		"uint32_t params_num; uint32_t line_idx; uint32_t file_name_len; "
 		"const char file_name[sizeof(\"src/debugability/macros.c\")]; "
 		"uint32_t text_len; const char text[sizeof(\"Message\")]; } "
-		"log_entry = { 1";
+		"log_entry = { 1(6 << 24)152sizeof(\"src/debugability/macros.c\")"
```

Comment:
I am not sure about this one(that `)152sizeof` seems sketchy), I made the change just to make unit tests shut up.
As @akloniex  pointed out to me this bug could be severe so in the scope of this PR I'd like to let this slide,
but I'll also want to sit down with @akloniex on what should be the correct value here as he will be modifying this very const char *should_be_eq here for https://github.com/thesofproject/sof/pull/530.

Reply:
@mwierzbix It seems like CAPTURE macro improperly removes commas from input. We'll look into that.

Reply:
Yeah, it's in fact the `CAPTURE` macro getting rid of `,`s. So it's a non-issue here.